### DISCUSSION TITLE
Error while loading Outlook Online after successful login

### DISCUSSION THREAD

**Discussion by ghost** [cypress-io/cypress#26709] [Component Testing]
Hi everyone 🙂 
I am trying to test an add-in for Outlook Online. Therefore I visit outlook.office.com and go through the authentication. After directing back to Outlook Online I get an error message from Outlook looking like this:
BootResult: cdnError
Failed resource: https://res.cdn.office.net/owamail/20230414002.09/scripts/owa.ven.graphql.m.js
Back Filled Errors: Unhandled Rejection: ChunkLoadError: Loading chunk 203918 failed.
(error: https://res-h3.public.cdn.office.net/owamail/20230414002.09/scripts/owa.ven.graphql.m.js):undefined%7CUnhandled Rejection: ChunkLoadError: Loading chunk 203918 failed.
...
(error: https://res.cdn.office.net/owamail/20230414002.09/scripts/owa.MailBoot.m.js):undefined%7CUnhandled Rejection: ChunkLoadError: Loading chunk 203918 failed.
(error: https://res.cdn.office.net/owamail/20230414002.09/scripts/owa.ven.graphql.m.js):undefined%7Cundefined:undefined%7Cundefined:undefined%7Cundefined:undefined%7Cundefined:undefined%7Cundefined:undefined%7Cundefined:undefined%7Cundefined:undefined%7Cundefined:undefined%7Cundefined:undefined%7Cundefined:undefined%7Cundefined:undefined%7Cundefined:undefined%7Cundefined:undefined%7Cundefined:undefined%7Cundefined:undefined
err: Error: Failed to load javascript.
esrc: Runtime
et: ClientError
estack: Error: Failed to load javascript.
    at P.onLoadFailed (https://outlook.office.com/mail/?bO=1&sessionId=07287aaf-f390-4e0b-8eb3-0783c4a9eedd:2:21272)
    at https://outlook.office.com/mail/?bO=1&sessionId=07287aaf-f390-4e0b-8eb3-0783c4a9eedd:2:20037

I use experimentalModifyObstructiveThirdPartyCode,  experimentalSessionAndOrigin and chromeWebSecurity: false

Did anyone happen to run into this issue and could be of any help here? 
Thanks a lot and if there are further questions please ask and i will try to answer them 🙂

**Comment by Martin-Held-PVG** [comment]
Hi, 
I have the same problem. Could you find a solution?
